src/index.html

```diff
@@ -21,7 +21,8 @@
     passageEndpoint: 'http://localhost:5001',
     environment: 'development',
     ingressBaseDomain: 'k8s.sample.io',
-    awsCapabilitiesJSON: '{"m3.large":{"description":"M3 General Purpose Large","memory_size_gb":7.5,"cpu_cores":2,"storage_size_gb":32},"m3.xlarge":{"description":"M3 General Purpose Extra Large","memory_size_gb":15,"cpu_cores":4,"storage_size_gb":80},"m3.2xlarge":{"description":"M3 General Purpose Double Extra Large","memory_size_gb":30,"cpu_cores":8,"storage_size_gb":160}}',
+    awsCapabilitiesJSON:
```

Comment:
Looks like this is from the other PR, lets make sure we don't merge it like this on accident.